getting_started/build-guide/build-from-scratch.rst

```diff
@@ -25,31 +25,44 @@ Set up build environment
 
 Install package dependencies.
 
-.. code-block:: bash
+ * For Ubuntu 18.10:
```

Comment:
Lines 28 and 38: There's an extra space here that is causing reStructuredText to interpret this as a block quote.